pkg/thing/delivery/amqp/client.go

```diff
@@ -25,6 +27,7 @@ type ClientPublisher interface {
 	SendUpdatedSchema(thingID string) error
 	SendDevicesList(things []*entities.Thing) error
 	SendAuthStatus(thingID string, errMsg *string) error
+	SendUpdateData(thingID string, data []network.Data) error
```

Comment:
By moving the `Data` structure to the `entities` folder this method will become similar to the others.

Reply:
Done!